hazardous/_gradient_boosting_incidence.py

```diff
@@ -84,6 +84,23 @@ def draw(self):
 class GradientBoostingIncidence(BaseEstimator, ClassifierMixin):
     """Cause-specific Cumulative Incidence Function (CIF) with GBDT.
```

Comment:
Use a "raw" docstring instead of a regular docstring to avoid having `\a` in `\approx` be interpreted as a special character at the Python source level.


```suggestion
    r"""Cause-specific Cumulative Incidence Function (CIF) with GBDT.
```